css/css-flexbox-1/negative-margins-001.html

```diff
@@ -4,11 +4,9 @@
 <link rel="author" title="Google Inc." href="https://www.google.com/">
 <link rel="help" href="http://www.w3.org/TR/css-flexbox-1/#item-margins">
 <link rel="match" href="reference/negative-margins-001-ref.html">
-<meta name="assert" content="Tests that for intrinsic size computations,
-an item will not take up less than zero space even with negative margins.">
 <style>
     .container {
-        width: 60px;
+        width: 40px;
```

Comment:
Why are you changing this to 40px?

Reply:
That's your change from the PR this is split out of. :)